src/components/layout/PanelChange/index.tsx

```diff
@@ -262,6 +267,49 @@ export const PanelChange: FC<IProps> = ({
 		onClickStop(callback);
 	}, [callback, onClickStop]);
 
+	const getIbAllouRatio = async (address: string, price: string) => {
+		const contract = await getContract(address, ibAllouToken, web3);
+		const ratio = await contract.methods.growingRatio().call();
+		const ratioFloat = await fromWei(ratio, 18);
+		const ratioPrice = parseFloat(ratioFloat) * parseFloat(price);
+		return ratioPrice.toString();
+	};
+
+	function isIbAlluo(coinA: Coin, coinB: Coin, totalFlow: string) {
```

Comment:
Can you improve the name of this function I'm not sure what it does by reading the name